project.clj

```diff
@@ -5,7 +5,7 @@
    :password :env/clojars_jenkins_password
    :sign-releases false})
 
-(defproject puppetlabs/analytics-client "1.0.3-SNAPSHOT"
+(defproject puppetlabs/analytics-client "1.1.0-SNAPSHOT"
```

Comment:
out of curiosity, why the Y bump?

Reply:
For the clj-parent updates. Although maybe they're not really sufficient to need it.

Reply:
ah, yeah, that makes sense. When I skimmed, I thought the clj-parent bump was a Z, but since it's a Y, I think a Y here makes sense. 👍🏼